oat-sa/browser-support-checker
# Feature/ADF-1417/Upgrade to Node 18

### Related to: [ADF-1417](https://oat-sa.atlassian.net/browse/ADF-1417)

### Summary

Update the devDependencies so that it installs seamlessly with Node 18.

### Details

To properly support Node 18, the dev dependencies had to be updated and aligned.

**Note:** The lock file was generated with `npm@6` so it would also work seamlessly with Node 14.

Several package has been upgraded to a major version:
- `@oat-sa/eslint-config-tao`: align the peer dependencies
- `babel-loader` drop of Node < 14.15 and require Babel >= 7.12
- `clean-webpack-plugin`: drop of Node 8 and webpack < 5
- `webpack-cli`: require webpack 5+

### How to test
- check the report on GitHub
- make sure to have Node 18 installed (you can use [nvm.sh](https://github.com/nvm-sh/nvm) for managing multiple versions)
- install the package: `npm i`
- build the bundle: `npm run build`
- generate the db: `npm update:browsers`

[ADF-1417]: https://oat-sa.atlassian.net/browse/ADF-1417?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
